### PR TITLE
Fixing Mef composition

### PR DESCRIPTION
**Pull request by jmarolf**
Fixing Mef composition error caused by previous commit
(11a827c4f7ea5cb72cfe559b7cc4664d59afa095)

### CODE REVIEW

**Comment by mavasani** [comment]
:+1: Thanks for the prompt fix Jon!

File: src/EditorFeatures/Core/Implementation/Workspaces/EditorErrorReportingService.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ internal class EditorErrorReportingService : IErrorReportingService
         public void ShowErrorInfoForCodeFix(string codefixName, Action OnEnableClicked, Action OnEnableAndIgnoreClicked)
         {
             var message = LogMessage.Create($"{codefixName} crashed");
-            Logger.Log(FunctionId.CodeFixe_Crash, message);
+            Logger.Log(FunctionId.Extension_Excpetion, message);
```
Comment (sharwell) [inline src/EditorFeatures/Core/Implementation/Workspaces/EditorErrorReportingService.cs]: :bulb: `Exception`?

Comment (jmarolf) [inline src/EditorFeatures/Core/Implementation/Workspaces/EditorErrorReportingService.cs]: good catch

**Comment by mavasani** [comment]
@jmarolf you may want to squash your commits.

**Pull request merged**